templates/dashboard/index.html.twig

```diff
@@ -27,7 +27,7 @@
                                           <tr {% for user in report.seen %}{% else %}class="table-warning"{% endfor %}>
                                             <th scope="row">{{ report.id }}</th>
                                             <td>{{ report.domain.fqdn }}</td>
-                                            <td>{{ report.organisation }}</td>
+                                            <td class="d-none d-md-table-cell">{{ report.organisation }}</td>
                                             <td>{{ report.dmarc_records.count }}</td>
```

Comment:
number of messages is actually not that important. its the end result that should be shown not the number of emails processed.

Reply:
Depends.
For my use case, "fqdn" and "organisation" is basically duplicated info so organization is not really required, but messages count is important to know because I look at reports from highest message count domains first.

Reply:
"fqdn" and "organisation" are different. FQDN is receiving party and Organisation is a sending party.

Reply:
OK, agreed. Also, the only real relevant info is whether there are any failures.
That would also solve the wrapping issue above, then we can use col-xl-6 without issues.